redesign/src/styles/elements/_elements.scss

```diff
@@ -0,0 +1,268 @@
+.footer {
```

Comment:
I'm not sure I understand the logic combining everything in this file. Can you describe what these elements/classes share in common? How do you know when something belongs here vs in the layout partials?

Reply:
The elements are supposed to be only focused on the type. So font size, weight, color, decoration and such.